### PR TITLE
Changes to schedule editor

### PR DESCRIPTION
**Pull request by vidya-ram**
Full width calendar.  
Add zoom buttons to change intervals

### CODE REVIEW

File: funnel/static/js/schedules.js [hunk 1]
```diff
@@ -515,7 +515,9 @@ $(function () {
         buttons.save = (function () {
           calendar.container
             .find('.fc-header-right')
-            .append('<span class="hg-fc-button save-schedule">Save</span>');
+            .append(
+              '<span class="hg-fc-button mui-btn save-schedule">Save</span>'
```
Comment (jace) [inline funnel/static/js/schedules.js:519]: Translation missing.

File: funnel/static/js/schedules.js [hunk 2]
```diff
@@ -550,6 +552,22 @@ $(function () {
             $(this).removeClass('fc-state-hover');
           }
         );
+      calendar.container
+        .find('.fc-header-left')
+        .append(
+          '<div class="tabs"><button class="tabs__item js-fc-zoom" data-slotinterval="5">5 mins</button> <button class="tabs__item js-fc-zoom" data-slotinterval="15">15 mins</button> <button class="tabs__item js-fc-zoom" data-slotinterval="30">30 mins</button> <button class="tabs__item js-fc-zoom" data-slotinterval="60">60 mins</button></div>'
```
Comment (jace) [inline funnel/static/js/schedules.js:558]: Translations missing.

File: funnel/static/js/schedules.js [hunk 3]
```diff
@@ -561,7 +579,7 @@ $(function () {
         calendar.container
           .find('.fc-header-right')
           .prepend(
-            '<label for="autosaver" class="fc-button fc-state-disabled fc-corner-right fc-corner-left"><input id="autosaver" class="autosave" type="checkbox"> Autosave</label> '
+            '<label for="autosaver" class="hg-fc-checkbox"><input id="autosaver" class="autosave" type="checkbox"> Autosave</label>'
```
Comment (jace) [inline funnel/static/js/schedules.js:582]: Translation.

File: funnel/templates/schedule_edit.html.jinja2 [hunk 4]
```diff
@@ -34,80 +34,83 @@
...
+          {%- for proposal in proposals['unscheduled'] %}
+            <div class="js-unscheduled proposal-box" data-modal-url="{{ proposal.modal_url }}">{{ proposal.title }}</div>
+          {%- endfor %}
+        </div>
+        <div id="settings" class="schedule-settings">
+          <form id="room_colors" action="javascript:void(0)" method="POST" accept-charset="UTF-8">
+            <div id="venue_sortable" class="sortable" data-drag-placeholder="ui-box-placeholder--bigger" draggable="true">
+            {% for venue in venues %}
+              <div class="ui-dragable-box ui-dragable-box--header">
+                <input name="venue-seq" value="{{ venue.seq }}" readonly class="seq-input seq-input--bg" data-venue="{{venue.uuid_b58}}">
```
Comment (jace) [inline funnel/templates/schedule_edit.html.jinja2:67]: Spacing in the Jinja2 tag: `{{venue.uuid_b58}}` -> `{{ venue.uuid_b58 }}`

File: funnel/templates/schedule_edit.html.jinja2 [hunk 5]
```diff
@@ -34,80 +34,83 @@
...
+                    {{ faicon(icon='angle-down', icon_size='caption', css_class='mui--text-light collapsible__icon collapsible__icon--disable') }}
+                    {{ faicon(icon='angle-up', icon_size='caption', css_class='mui--text-light collapsible__icon collapsible__icon--disable mui--hide') }}
+                  </div>
+                  <div id="rooms_sortable" class="sortable collapsible__body collapsible__body--disable" data-drag-placeholder='ui-box-placeholder'>
+                    {% for room in venue.rooms %}
+                      <p class="ui-dragable-box" draggable="true">
+                        {{ faicon(icon='grip-vertical', icon_size='caption', baseline=false, css_class='mui--text-light drag-icon') }}
+                        <input name="room-seq" value="{{ room.seq }}" readonly class="seq-input" data-room="{{room.uuid_b58}}">
+                        <label for="room-color-input-{{ room.uuid_b58 }}" class="h5">
+                          <input type="hidden" name="room-uuidb58" value="{{ room.uuid_b58 }}" data-venue="{{venue.uuid_b58}}">
```
Comment (jace) [inline funnel/templates/schedule_edit.html.jinja2:82]: Spacing again (this is just a stylistic convention).

File: funnel/templates/schedule_edit.html.jinja2 [hunk 6]
```diff
@@ -133,7 +136,7 @@
       var ROOMS = {{ rooms|tojson }};
       var BREAK_EVENTS_COLOR = "#999999";
       var EDIT_EVENTS = true;
-      var CALENDAR_SLOT_MINUTES = 5;
+      var CALENDAR_SLOT_MINUTES = 30;
```
Comment (jace) [inline funnel/templates/schedule_edit.html.jinja2:139]: Is there a way to manually edit times to choose a smaller interval? Zainab loves her 5 minute sessions.

**Review by jace — CHANGES_REQUESTED** [review]

File: funnel/templates/schedule_edit.html.jinja2 (hunk 6 above)
Comment (vidya-ram) [reply funnel/templates/schedule_edit.html.jinja2:139]: Have added buttons to switch between 5m, 15m, 30m, 60m intervals

File: funnel/static/js/schedules.js (hunk 1 above)
Comment (vidya-ram) [reply funnel/static/js/schedules.js]: Updated babel.cfg to extract from .js files.

**Review by jace — CHANGES_REQUESTED** [review]
Some grammar issues in UI text (yet to review for logic).

**Pull request merged**